help-view/bnd.bnd

```diff
@@ -1 +1,2 @@
+Import-Package: !java.*,*
```

Comment:
Is it checked that the manifest contains this package import and it works if it's explicitely excluded ?

Reply:
I think I checked the generated headers and I'm sure I tested it with Java 11. Don't know if it is backwards compatible with Java 8.

Reply:
> As mentioned in the related ticket, I can't currently work with the master branch so I had to do it here instead. I guess we can then cherry pick.

Sorry , can't find it in the ticket .
Could you please give a link ?

Reply:
> I think I checked the generated headers and I'm sure I tested it with Java 11. Don't know if it is backwards compatible with Java 8.

Does it mean:
- without this change and Java 11 config the MANIFEST contains extra `java.*`  imports ?
- with this changes the MANIFEST does contain extra imports ?
- both ?

For me with Java 11 and without `maven.compiler.source`  the manifest doesn't contain extra headers.
So I cannot reproduce the issue and that's why I'm asking: does it fix anything at all ?

Reply:
Oh, I mixed with the #27 and a branch related to that where I have the same problem.

Yes, with this branch, that has Java 11 set for compilation, if compiled with JDK and without those bnd hints, bad things will happen. With the hints they don't.  We actually have similar for the main UI bundle:

https://github.com/vaadin/base-starter-flow-karaf/blob/v19/starter/bnd.bnd#L1

Reply:
The ticket header is : 
`can't start greetservice in Vaadin 20 alpha 7 if Java 11 language features are turned on` .

I'm not familiar with the term "Java 11 language features", sorry.
It's not clear for me how Maven compiler settings may be matched to "Java 11 language feature".

The point which I'm trying to ask for the ticket reporters: there should be no any misunderstanding (/assumptions).
The ticket has to have all the necessary information.
The ticket has to have 
```
<maven.compiler.source>11</maven.compiler.source>
<maven.compiler.target>11</maven.compiler.target>
```
snippet in its description.

Such description would have saved a lot of time

Reply:
In the description there was also a link to a branch with this change: https://github.com/vaadin/base-starter-flow-karaf/commit/8c8abe2ebddd8591efc5d959206289945f46c1f5 

We are all doing our best, no need to get frustrated.

Reply:
OK, that's my bad, indeed.
Even though I would prefer to see the only relevant info printed directly in the ticket (desirably in bold).
But anyway, sorry: I guess you modified the ticket and I missed the modification since it's hard to see the changes 
and there were no reasons to reread the description which looked the same.

Still .
My original point is the same: 
- I spent more than half of a day in attempt to reproduce the issue.
- With the information about pom modification with two properties `maven.compiler.source`  and `maven.compiler.target` I would have spent 5 minutes to reproduce and 5 minutes to make a PR.

The time difference is so huge and there is no any point to not provide the most important information right away avoiding any confusion.
For some reason it happens all the time.